scopes/dependencies/pnpm/lynx.ts

```diff
@@ -226,6 +229,23 @@ export async function install(
   }
 }
 
+async function includeAllComponentsFromDir (rootDir: string, manifestsByPaths) {
+  const files = await fs.readdir(rootDir, { withFileTypes: true })
+  return Promise.all(
+    files
+      .filter((file) => file.isDirectory() && file.name !== 'node_modules')
+      .map((dir) => path.join(rootDir, dir.name))
+      .filter((dirPath) => !manifestsByPaths[dirPath])
```

Comment:
add another filter to check if there is a pacakge.json file.
We create some other folders in the root sometimes which are not capsules, and won't have package.json

Reply:
it is covered at line 241.

if the `package.json` is not found, such error is ignored at line 243